packages/sygma-ui/package.json

```
@@ -7,14 +7,13 @@
   "dependencies": {
     "@babel/core": "^7.12.3",
     "@babel/runtime": "^7.12.1",
+    "@buildwithsygma/sygma-contracts": "1.0.0",
+    "@buildwithsygma/sygma-sdk-core": "1.1.2",
```

Please use 1.1.3 version here
```suggestion
    "@buildwithsygma/sygma-sdk-core": "1.1.3",
```